### ISSUE TITLE
DPL-nnn Transfer PBMCs from tube rack to plate

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/limber#1483]
**User story**
As a user of the scRNA Core Donor Pooling pipeline, I would like the LIMS to support the transfer from the LRC PBMC Cryostor tube rack into the LRC PBMC Defrost PBS plate, along with bed verification to help prevent human error.

**Who are the primary contacts for this story**
Lesley, Abby, Andrew, Katy

**Who is the nominated tester for UAT**
Abby, Lesley

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Input labware is tube rack, as represented by https://github.com/sanger/limber/issues/1388
- [ ] Output labware is LRC PBMC Defrost PBS
- [ ] Straight stamp
- [ ] Bed verification details TBC

**Dependencies**
This story is blocked by the following dependencies:
- https://github.com/sanger/limber/issues/1388

**Comment by SujitDey2022** [comment]
@KatyTaylor split the bed verification acceptance criteria into a new story https://github.com/sanger/limber/issues/1577

**Comment by yoldas** [comment]
Except integration suite changes, there is nothing to do in Limber  because the StampedPlate is already the default labware creator, pipeline configuration was done, well filter skips the empty and failed wells,  and the bed verification story is separated and not required in MVP.
